test/integration/admin_chips_test.rb

```diff
@@ -47,4 +47,41 @@ def create_admin
     assert_equal admin_chips_path, current_path
     assert page.has_content? 'EditedName'
   end
+
+  test 'admin can add chip' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.